src/main/java/blusunrize/immersiveengineering/common/util/compat/OneProbeCompatModule.java

```diff
@@ -0,0 +1,356 @@
+			{
+				SheetmetalTankTileEntity master = ((SheetmetalTankTileEntity) tileEntity).master();
+				int current = master.tank.getFluidAmount();
+				int max = master.tank.getCapacity();
+
+				if(current > 0)
+				{
+					probeInfo.progress(current, max,
+							probeInfo.defaultProgressStyle()
+									.suffix("mB")
+									.numberFormat(NumberFormat.COMPACT));
+				}
+			}
+		}
+	}
+
+
+	public static class EnergyInfoProvider implements IProbeInfoProvider, IProbeConfigProvider
+	{
+
+		@Override
+		public String getID()
+		{
+			return ImmersiveEngineering.MODID+":"+"EnergyInfo";
+		}
+
+		@Override
+		public void addProbeInfo(
+			ProbeMode mode,
+			IProbeInfo probeInfo,
+			PlayerEntity player,
+			World world,
+			BlockState blockState,
+			IProbeHitData data
+		)
+		{
+			TileEntity tileEntity = world.getTileEntity(data.getPos());
+			int cur = 0;
+			int max = 0;
+			if (tileEntity instanceof IFluxReceiver)
+			{
+				cur = ((IFluxReceiver) tileEntity).getEnergyStored(null);
+				max = ((IFluxReceiver) tileEntity).getMaxEnergyStored(null);
+			}
+			else if (tileEntity instanceof IFluxProvider)
+			{
+				cur = ((IFluxProvider) tileEntity).getEnergyStored(null);
+				max = ((IFluxProvider) tileEntity).getMaxEnergyStored(null);
+			}
+			if (max > 0)
+			{
+				probeInfo.progress(cur, max,
+						probeInfo.defaultProgressStyle()
+								.suffix("IF")
+								.filledColor(Lib.COLOUR_I_ImmersiveOrange)
+								.alternateFilledColor(0xff994f20)
+								.borderColor(Lib.COLOUR_I_ImmersiveOrangeShadow)
+								.numberFormat(NumberFormat.COMPACT));
+			}
+		}
+
+		@Override
+		public void getProbeConfig(
+			IProbeConfig config,
+			PlayerEntity player,
+			World world,
+			Entity entity,
+			IProbeHitEntityData data
+		)
+		{
+		}
+
+		@Override
+		public void getProbeConfig(
+			IProbeConfig config,
+			PlayerEntity player,
+			World world,
+			BlockState blockState,
+			IProbeHitData data
+		)
+		{
+			TileEntity tileEntity = world.getTileEntity(data.getPos());
+			if(tileEntity instanceof IFluxReceiver||tileEntity instanceof IFluxProvider)
+			{
+				config.setRFMode(0);				
+			}
+		}
+	}
+
+	public static class ProcessProvider implements IProbeInfoProvider
+	{
+
+		@Override
+		public String getID()
+		{
+			return ImmersiveEngineering.MODID+":"+"ProcessInfo";
+		}
+
+		@Override
+		public void addProbeInfo(
+			ProbeMode mode,
+			IProbeInfo probeInfo,
+			PlayerEntity player,
+			World world,
+			BlockState blockState,
+			IProbeHitData data
+		)
+		{
+			TileEntity tileEntity = world.getTileEntity(data.getPos());
+			if(tileEntity instanceof IEBlockInterfaces.IProcessTile)
+			{
+				int[] curTicks = ((IEBlockInterfaces.IProcessTile) tileEntity).getCurrentProcessesStep();
+				int[] maxTicks = ((IEBlockInterfaces.IProcessTile) tileEntity).getCurrentProcessesMax();
+				int h = Math.max(4, (int)Math.ceil(12/(float)curTicks.length));
+	
+				for(int i = 0; i < curTicks.length; i++)
+				{
```

Comment:
Parenthesis for this loop are not necessary, since it only contains the condition